recipes/raxml/meta.yaml

```diff
@@ -22,15 +22,17 @@ test:
     - raxmlHPC-PTHREADS -h > /dev/null
     - raxmlHPC-SSE3 -h > /dev/null
     - raxmlHPC-PTHREADS-SSE3 -h > /dev/null
-    - raxmlHPC-AVX2 -h > /dev/null
-    - raxmlHPC-PTHREADS-AVX2 -h > /dev/null
+#    - '[ "$(uname -m)" != "aarch64" ] && raxmlHPC-AVX2 -h > /dev/null'
```

Comment:
Is there a way to make some of the commands conditional ?
Here I need to execute the AVX tests only on x86_64. 
The approach with `'[ "$(uname -m)" != "aarch64" ] && ...` works for the "simple" test execution but fails for the mulled test because all the commands are `and`ed (command1 && command2 && ...) and this causes the mulled test to fail on Linux ARM64.

Reply:
Thank you, @mencian !